docker/development/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM php:7.4-cli-alpine
```

Comment:
Tip: Maak hier een multistage Dockerfile van, je kan dan de productie laag van de image hergebruiken voor development.

Voorbeeld:

FROM php:7.4-cli-alpine AS production

# productie dingen hier

FROM production AS development

# hier development dingen toevoegen

Reply:
Handy! Ik zag het ook in Dockerfile in de app root voorbij komen, ben niet bekend met het multistage concept binnen Docker maar sounds logic :) Kan dit dan ook in de Dockerfile geplaatst die in de app root staat?

Reply:
Ik heb nu een development stage toegevoegd in de Dockerfile:

```
##################################################################################################################
# Development Stage
##################################################################################################################
FROM production AS dev

RUN composer install
```

Reply:
Zeker! Je zal dan alleen tijdens het bouwen van bijv. de productiebuild moeten aangeven dat je tot een specifiek target wil gaan en niet verder.

Reply:
Heb nu een development stage (helemaal onderaan) toegevoegd `FROM base_image AS development` in de Dockerfile in de app root. Hier staan verder (nog) geen commands in.
In `docker/development/docker-compose.yml` heb ik het volgende nu staan mbt de build:
```
    build:
      context: ../../
      dockerfile: Dockerfile
      target: development
```

Als ik echter daarna `docker-compose --build` doe, zou ik verwachten dat er een development image wordt gebouwd voor de service, zonder de test en production_build stages. Toch lijkt dit niet te gebeuren, want ik zie bij het builden de output van test voorbij komen.

Hetzelfde als ik `docker build --target development -t jorijn/bitcoin-dca:development .` uitvoer.